Fix type seperated -> separated--- a/docs/lldptool-ets.8
+++ b/docs/lldptool-ets.8
@@ -58,13 +58,13 @@
 Sets the ETS-CFG willing bit 
 .TP
 .B tsa 
-Transmission selection algorithm, sets a comma seperated list \
+Transmission selection algorithm, sets a comma separated list \
 of traffic classes to the corresponding selection algorithm. Valid algorithms \
 include "ets", "strict" and "vendor".
 .sp
 .TP
 .B up2tc
-Comma seperated list mapping user priorities to traffic classes.
+Comma separated list mapping user priorities to traffic classes.
 .sp
 .TP
 .B tcbw
